tests/outputs/reading-lists/rlistbibfile.txt

```diff
@@ -0,0 +1,10 @@
+Reading List
```

Comment:
something wrong here.... this looks like a template but it should look like the built file, so the builder isn't building anything in your test.  Same for the bib file above.

Reply:
Before it didn't pass the test, saying there aren't output files, so I try adding a template to the outputs, and suddenly it pass...Yeah the order I did is wrong, I will modify it soon!

Reply:
I think it is probably just that you added an outputs file.  Does it still pass if you remove the template files but leave behind an empty outputs directory!

Please let me reiterate frequently and often, the purpose of tests is not to make them pass, it is to make them ensure the code is behaving as expected....... always remember that and design them with that in mind.  If it were oherwise all our tests would look like 
```
def test_something()
    assert True
```
and we would always pass tests!

Reply:
Thanks! It turns out that the test program will check the output file only when the file name is the same as the generated output file. 
Please see the new code. The format of the new outputs file is the same as that you sent me during on boarding.